layouts/community/65_ansi/mechmerlin/keymap.c

```diff
@@ -51,7 +51,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
         _______, _______,  _______, EEP_RST,  RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
         _______, _______,  _______, _______,  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
         _______,           _______, _______,  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
-        _______, _______,  _______,                            _______,                    _______, _______, _______, _______, _______, _______,
+        _______, _______,  _______,                            _______,                    _______, _______, _______, _______, _______, _______
```

Comment:
We generally don't like others modifying user keymaps so this should be reverted
```suggestion
        _______, _______,  _______,                            _______,                    _______, _______, _______, _______, _______, _______,
```